playground/modular/entry.js

```diff
@@ -0,0 +1,102 @@
+import ListBase from 'devextreme/ui/list_light';
+
+const container = document.getElementById('container');
+const newElement = (name) => {
+    const title = document.createElement('p');
+    title.innerText = name;
+    const el = document.createElement('div');
+    container.appendChild(title);
+    container.appendChild(el);
+    return el;
+};
+
+import 'devextreme/ui/list/modules/selection';
+new ListBase(newElement('selection'), {
+    dataSource: [ 1, 2, 3 ],
+    showSelectionControls: true,
+    selectionMode: 'single'
+});
+
+import 'devextreme/ui/list/modules/deleting';
+new ListBase(newElement('deleting'), {
+    dataSource: [ 1, 2, 3 ],
+    allowItemDeleting: true
+});
+
+import 'devextreme/ui/list/modules/deleting.context';
+new ListBase(newElement('itemDeleteMode context'), {
+    dataSource: [ 1, 2, 3 ],
+    allowItemDeleting: true,
+    itemDeleteMode: 'context'
+});
+
+import 'devextreme/ui/list/modules/deleting.swipe';
+new ListBase(newElement('itemDeleteMode swipe'), {
+    dataSource: [ 1, 2, 3 ],
+    allowItemDeleting: true,
+    itemDeleteMode: 'swipe'
+});
+
+import 'devextreme/ui/list/modules/deleting.slideItem';
+new ListBase(newElement('itemDeleteMode slideItem'), {
+    dataSource: [ 1, 2, 3 ],
+    allowItemDeleting: true,
+    itemDeleteMode: 'slideItem'
+});
+
```

Comment:
I think explicit syntax will look better

Reply:
Don't forget that the Vanilla JS is not basic scenario of usage. We'll put this dependency to the corresponding nested components for Angular, React and Vue.